### PR TITLE
Synced files with skyux library, updated docs, fix for sort issue on list view grid

### PR DESCRIPTION
**Pull request by blackbaud-joshgerdes**

### CODE REVIEW

File: src/app/learn/tutorials/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        </li>
+        <li><a routerLink="/learn/tutorials/host">Host a project</a></li>
+    </ul>
+
+<sky-alert alertType="info">These tutorials currently only cover the basics. We will create additional tutorials as we identify areas where they are necessary.</sky-alert>
+
+        <p><strong>Next step »</strong> <a routerLink="/learn/tutorials/create-project">Create a project</a></p>
+    <p><strong>Previous step »</strong> <a routerLink="/learn">Get started</a></p>
+
+</div>
```
Comment (Blackbaud-LuisBello) [inline src/app/learn/tutorials/index.html]: newline

File: src/app/learn/tutorials/install/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <ul>
+        <li><p>To find the latest version of the SKY UX components, check the <a href="https://github.com/blackbaud/skyux2/blob/master/CHANGELOG.md">changelog in the SKY UX 2 repo</a> or the <a href="https://www.npmjs.com/package/@blackbaud/skyux">NPM page for the SKY UX 2 package</a>.</p></li>
+        <li><p>To find the latest version of SKY UX Builder, check the <a href="https://github.com/blackbaud/skyux-builder/blob/master/CHANGELOG.md">changelog in the SKY UX Builder repo</a> or the <a href="https://www.npmjs.com/package/@blackbaud/skyux-builder">NPM page for the SKY UX Builder package</a>.</p></li>
+    </ul>
+<sky-alert alertType="info">If you use VSCode or another editor with code-editing functionality, you can also open a prompt that displays the latest version by placing the cursor in the property and pressing Ctrl-Space.</sky-alert>
+    
+    <p><strong>Next step »</strong> <a routerLink="/learn/tutorials">Tutorials</a></p>
+    <p><strong>Previous step »</strong> <a routerLink="/learn">Get started</a></p>
+
+</div>
```
Comment (Blackbaud-LuisBello) [inline src/app/learn/tutorials/install/index.html]: newline

File: .jscsrc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "preset": "airbnb",
```
Comment (blackbaud-joshgerdes) [inline .jscsrc]: it's a thing: https://github.com/jscs-dev/node-jscs/blob/master/presets/airbnb.json

basically, just synced skyux team's repo with ours to assure we can merge our code. Some items might not be correct or how we'd do it. ;)

File: src/app/learn/tutorials/install/index.html (hunk 2 above)
Comment (blackbaud-joshgerdes) [inline src/app/learn/tutorials/install/index.html]: that's their html, not important to this repo

File: src/app/learn/tutorials/index.html (hunk 1 above)
Comment (blackbaud-joshgerdes) [inline src/app/learn/tutorials/index.html]: that's their html, not important to this repo

**Pull request merged**